kotlin/src/main/kotlin/io/logto/sdk/core/type/IdTokenClaims.kt

```diff
@@ -0,0 +1,10 @@
+package io.logto.sdk.core.type
+
+data class IdTokenClaims(
+    val sub: String,
+    val atHash: String,
```

Comment:
maybe optional according to RFC and slack discussion. pending finalization on Slack.

Reply:
updated